shared/message/platform/darwin/DarwinSensors.proto

```diff
@@ -44,115 +38,40 @@ message DarwinSensors {
         CORRUPT_DATA  = 128;
         TIMEOUT       = 256;
     }
-
-    /**
-     * Toggle status of the LEDs on the panel.
-     */
     message LEDPanel {
-
-        /**
-         * led2 toggle status.
-         */
         bool led2 = 1;
-
-        /**
-         * led3 toggle status.
-         */
         bool led3 = 2;
-
-        /**
-         * led4 toggle status.
-         */
         bool led4 = 3;
     }
-
-    /**
-     * The colour for the HeadLED to display.
-     */
     message HeadLED {
-
-        /**
-         * Encode as 0x00 0xRR 0xGG 0xBB.
-         */
+        /// Encode as 0x00 0xRR 0xGG 0xBB
         uint32 RGB = 1;
     }
-
-    /**
-     * The colour for the EyeLED to display.
-     */
     message EyeLED {
-
-        /**
-         * Encode as 0x00 0xRR 0xGG 0xBB.
-         */
+        /// Encode as 0x00 0xRR 0xGG 0xBB
         uint32 RGB = 1;
     }
-
-    /**
-     * Press-down status of left and middle buttons.
-     */
     message Buttons {
-
-        /**
-         * Press-down status of the left button.
-         */
-        bool left = 1;
-
-        /**
-         * Press-down status of the middle button.
-         */
+        bool left   = 1;
         bool middle = 2;
     }
-
-    /**
-     * Raw accelerometer values from the CM740: x, y, z.
-     */
+    /// Raw accelerometer values from the CM740: {x, y, z}
+    /// The value is between 0 and 1023, representing a value between -4g and 4g => 512 = 0g
     message Accelerometer {
-
-        /**
-         * Raw x acceleration value from the CM740.
-         */
         float x = 1;
-
-        /**
-         * Raw y acceleration value from the CM740.
-         */
         float y = 2;
-
-        /**
-         * Raw z acceleration value from the CM740.
-         */
         float z = 3;
     }
-
-    /**
-     * The raw gyro values from the CM740 (they are read from the board in ZYX): x, y, z.
-     */
+    /// The raw gyro values from the CM740 (they are read from the board in ZYX)
```

Comment:
You should add information on the range of values, same as you did for the accelerometer